app/templates/rules.html

```diff
@@ -84,7 +84,12 @@ <h3 class="panel-title">
     <div class="left" style="float: left">
 
       <h5 title="Mapping">
-        Mapping: <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+        Mapping: <span ng-show="rule.mapping">
+            <b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b>
+            <!--<a href="/releases/#{{ rule.mapping }}"><b ng-bind-html="highlightSearch(rule.mapping, 'mapping')"></b></a>-->
+            <a href="" ng-show="rule.mapping" ng-click="openReleaseDataModal(rule.mapping)" class="glyphicon glyphicon-eye-open" title="View"></a>
```

Comment:
I think it's much better to us a button. Two reasons;
1. You don't need to have that hackish `href=""` thing. 
2. Buttons are more user-friendly. They're bigger. They have text and not just an icon. 
You can still have an icon *inside* the button. E.g.
```html
<a class="btn btn-default" href="" ng-click="openReleaseDataModal(rule.mapping)">
    <i class="glyphicon glyphicon-eye-open"></i>
    View
</a>
```
Or better;
```html
<button class="btn btn-default" ng-click="openReleaseDataModal(rule.mapping)">
    <i class="glyphicon glyphicon-eye-open"></i>
    View
</button>
```

Check out: http://getbootstrap.com/css/#buttons
You can play with the size and the color.